packages/expo-symbols/package.json

```diff
@@ -33,7 +37,9 @@
   "license": "MIT",
   "homepage": "https://docs.expo.dev/versions/latest/sdk/symbols/",
   "dependencies": {
-    "sf-symbols-typescript": "^2.0.0"
+    "sf-symbols-typescript": "^2.0.0",
+    "@expo-google-fonts/material-symbols": "^0.4.1",
```

Comment:
I don't like this direct dependency, but peer deps don't seem to work for building the module, so it's either that or ask the user to manually add these packages.

The second option might be safer (?)